### ISSUE TITLE
Bot for updating dependencies?

### ISSUE THREAD

**Issue by martin-sucha** [nsqio/nsq#1320]
Hello,

have you considered adding integration with [Dependabot](https://dependabot.com/go/) or [Renovate bot](https://docs.renovatebot.com/) to automatically open pull requests to update dependencies?

**Comment by mreiferson** [comment]
No, I've typically looked to update dependencies as part of stamping a new release. I suppose I'd consider integration with dependabot.
